test/libsolidity/semanticTests/externalContracts/strings.sol

```diff
@@ -51,9 +51,9 @@ contract test {
 // compileViaYul: also
 // ----
 // constructor()
-// gas irOptimized: 912777
+// gas irOptimized: 910871
 // gas legacy: 1188228
-// gas legacyOptimized: 771634
+// gas legacyOptimized: 749336
```

Comment:
How did this change?

Reply:
Ah, because of abi code?

Reply:
Hm... but that doesn't go through the IR code generator, does it? Without thinking about it, this seems rather weird to me, too...

Reply:
I ran the test setting a breakpoint on the change and `compileViaYul: false`. It doesn't enter there. So this is indeed weird.

Reply:
Might be worth figuring out what's going on there, then... my guess would be that it's either a bug in how ``isoltest`` calculates gas costs or some weird bug, s.t. whatever happened before the ``legacyOptimized`` run changes its behaviour (e.g. some change in inlining order in the abi code due to some state bleeding through or AST IDs changing - respectively *something* like that - I don't really see what kind of thing along those lines could *actually* be happening here :-)).